### PR TITLE
qesap AWS config file

### PR DESCRIPTION
**Pull request by mpagot**
Remove quotes from region in all scenarios
Store aws config file deployed in the cluster
Add unit testing

Related ticket: TEAM-7152

Verification run:
 -  http://openqaworker15.qa.suse.cz/tests/140823 and here an example of the file http://openqaworker15.qa.suse.cz/tests/140823/logfile?filename=test_cluster-vmhana01-aws_config.txt

### CODE REVIEW

File: lib/qesapdeployment.pm [hunk 1]
```diff
@@ -553,7 +553,8 @@ sub qesap_create_aws_credentials {
 sub qesap_create_aws_config {
     my %paths = qesap_get_file_paths();
     my $region = script_output q|awk -F ' ' '/aws_region/ {print $2}' | . $paths{qesap_conf_trgt};
-    $region = get_required_var('PUBLIC_CLOUD_REGION') if ($region =~ /^%.+%$/);
+    $region = get_required_var('PUBLIC_CLOUD_REGION') if ($region =~ /^["']?%.+%["']?$/);
+    $region =~ s/[\"\']//g;
```
Comment (alvarocarvajald) [inline lib/qesapdeployment.pm:557]: If you move this line one up, the regexp in the condition in L556 gets easier.

**Review by alvarocarvajald — APPROVED** [review]
LGTM. Just a small comment.

File: lib/qesapdeployment.pm (hunk 1 above)
Comment (mpagot) [reply lib/qesapdeployment.pm:557]: but I like it also to take care to remove quotes eventually present in the `PUBLIC_CLOUD_REGION` setting

**Review by lpalovsky — APPROVED** [review]
LGTM

**Pull request merged**